apps/envited-x-data-space/common/fetcher/fetcher.ts

```diff
@@ -0,0 +1,5 @@
+export const fetchAssetDataByCID = async (cid: string) => {
```

Comment:
This file shouldnt be called fetcher, thats for the http implementation itself. Call it API or something.